ITE/7/README.adoc

```diff
@@ -236,20 +241,22 @@ of verification could be added to the layout to alleviate this concern.
 [[infrastructure-requirements]]
 == Infrastructure Requirements
 
-If your changes require additional infrastructure, describe it here. Include
-potential costs incurred considering both time and money.
+Currently the only additional infrastructure being used in support of this ITE
+are test SPIRE servers being started during the testing pipeline.
 
 [[testing]]
 == Testing
 
-Our prototype implementation includes some basic unit and integration/system
-testing to ensure our prototype works.  More tests can be created.
+Currently tests exist within in-toto-golang that start a test SPIRE server and
+use short lived certificates to sign link metadata and use the signed metadata
+to verify an in-toto layout.
 
 [[prototype-implementation]]
 == Prototype Implementation
 
-A current proof-of-concept implementation of this ITE exists at Boxboat's fork
-of the in-toto-golang project: https://github.com/boxboat/in-toto
+A prototype implementation of this ITE was merged into the main branch of
+in-toto-golang as part of
+link:https://github.com/in-toto/in-toto-golang/pull/119[Pull Request 119]
 
 [[references]]
 == References
```

Comment:
We should expand this to point to resources for X.509, SPIFFE/SPIRE, maybe Fulcio as well if we add that as a use case. What do you think?

Reply:
That's a good idea, I will get this updated asap